Jenkinsfile

```diff
@@ -14,7 +14,7 @@ pipeline {
                     }
                     steps {
                         sh 'sudo /bin/bash -c "source ./script/installation/packages.sh"'
-                        sh 'python ./script/validators/source_validator.py'
+                        sh 'python3 -m script.validators.source_validator'
```

Comment:
Why do we want to do this?

Reply:
Since `script` is now a Python package, this is the correct way to access the internal scripts.